src/modules/buyNFT/components/InfoTabs/InfoTabsSkeleton.tsx

```diff
@@ -0,0 +1,46 @@
+import { makeStyles, Theme } from '@material-ui/core';
+import React from 'react';
+import {Skeleton} from "@material-ui/lab";
+
+const useInfoTabsSkeletonStyles = makeStyles<Theme>(theme => ({
+  tabs: {
+    display: "flex",
+    flexFlow: "row nowrap",
+    marginBottom: "15px",
```

Comment:
let's avoid units when possible and round to basic size if it's just one pixel
```suggestion
    marginBottom: theme.spacig(2),
```